### PR TITLE
Do not forward incoming VXLAN packets.

### PR DESCRIPTION
**Pull request by fasaxc**
## Description

Disable the FIB when we detect a VXLAN packet to the local host.  The FIB lookup is not helpful and on that code path the FIB input datastructure hasn't be initialised so we end up sending the packet to the default gateway.

Bug was introduced by interaction between #2666 and #2669.  Only seems to show up in E2Es; perhaps the FIB is failing for another reason in the FVs?

## Todos
- [ ] Unit tests (full coverage)
- [ ] Integration tests (delete as appropriate) In plan/Not needed/Done
- [ ] Documentation
- [ ] Backport
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Review by neiljerram — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
